crypto/threads_pthread.c

```diff
@@ -59,7 +612,7 @@ CRYPTO_RWLOCK *CRYPTO_THREAD_lock_new(void)
     pthread_mutexattr_t attr;
     CRYPTO_RWLOCK *lock;
 
-    if ((lock = CRYPTO_zalloc(sizeof(pthread_mutex_t), NULL, 0)) == NULL)
+    if ((lock = OPENSSL_zalloc(sizeof(pthread_mutex_t),)) == NULL)
```

Comment:
Comma useless in `sizeof(pthread_mutex_t) , ))`
Is this code compiled ?

Reply:
highly unlikely given that USE_RWLOCK should be almost always defined.  Also I'll fix it here, but note this is outside the functionality that this PR is adding.

Reply:
@nhorman I don't see the fixup for this and a few other comments of mine.
Forget to push a commit ?